src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,79 +24,94 @@ redirects:
   - /docs/alerts-applied-intelligence
 ---
 
-import accountsIssueFeedforOverviewPage from 'images/accounts_screenshot-full_issue-feed-for-overview-page.png'
+import alertsAlertsandAiOverviewPage from 'images/alerts_screenshot-full_alerts-and-ai-overview-page.png'
 
-Alerts and applied intelligence are a suite of tools and features that proactively notify you when things go wrong... right when they're starting to go wrong.
+As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening where, and what to do about it. New Relics alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
 
-Want to try Alerts & AI? Join us! It's free... forever!
+Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that alerts your team to potential problems by first identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues and providing a root cause analysis.
 
-<ButtonGroup>
-  <ButtonLink
-    role="button"
-    to="https://newrelic.com/signup"
-    variant="primary"
-  >
-    Sign up for free
-  </ButtonLink>
-</ButtonGroup>
+Alerts and Applied intelligence were created for busy DevOps teams: it reduces noise and makes sure you never miss an issue. 
 
-Already have an account? [Login](https://one.newrelic.com).
+## How it all works 
 
-Alerts lets you define the conditions in your system you want to be notified about, letting you focus on the things you care about most.
+### Detect and diagnose potential problems in your system [#detect-intro]
 
-Applied intelligence is a machine-learning engine that reduces alert noise, correlates incidents, automatically detects anomalies, and provides root cause analysis. Not only that, it enriches your notifications with metadata to give you greater context of what's happening when you are notified.
+New Relic never stops observing your telemetry data and we created alerts so your team can decide when you want to be notified and how.  Applied intelligence takes alerting one step further and monitors for any deviations from what's expected and notifies your team instantly. You can decide what level of automation works best for each situation. Our automatic anomaly detection is easy to use and implement, while our more customizable detection options offer your team the ability to create alerts that are just right for you. 
+
+Learn how to configure anomalies [in our anomaly docs](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
```

Comment:
```suggestion
Learn how to configure anomalies [in our anomaly docs](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
```

Reply:
I think this is the wrong doc to link to, here. Maybe this is a better link? https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence/#there-are-two-types-of-anomaly-detection-automatic-and-custom